ush/rocoto/setup_workflow.py

```diff
@@ -1334,19 +1516,41 @@ def create_xml(dict_configs):
         dict_hyb_tasks = get_hyb_tasks(dict_configs)
 
         # Removes <memory>&MEMORY_JOB_DUMP</memory> post mortem from hyb tasks
-        hyp_tasks = {'gdaseobs':'gdaseobs',
-                     'gdasediag':'gdasediag',
-                     'gdaseomg':'gdaseomn',
-                     'gdaseupd':'gdaseupd',
-                     'gdasecen':'gdasecmn',
-                     'gdasesfc':'gdasesfc',
-                     'gdasefcs':'gdasefmn',
-                     'gdasepos':'gdasepmn',
-                     'gdasearc':'gdaseamn',
-                     'gdasechgres':'gdasechgres'}
+        do_efsoi = base.get('DO_EFSOI', 'NO').upper()
+
+        if do_efsoi in ['Y', 'YES']:
+            hyp_tasks = {'gdaseobs':'gdaseobs',
```

Comment:
This variable doesn't appear to be used in active code, perhaps we should just remove it.

Reply:
@WalterKolczynski-NOAA Which variable do you mean?

Reply:
Nevermind, I didn't look far enough down behind the cut for it.